M2/BUILD/mahrud/Makefile

```diff
@@ -0,0 +1,24 @@
+# 
+# Written for Fedora 26
+#
+
+.PHONY: always
+
+rpm-docker-build:
+	[ ! -z `docker images -q m2-rpm` ] || docker build -t m2-rpm .
+	docker run -it \
+	  -v `pwd`/../../..:/M2 \
+	  m2-rpm \
+	  make rpm-build -j4 -C /M2/M2/BUILD/mahrud
+
+rpm-build:
+	mkdir -p build;
+	make -j4 -C ../..;
+	cd build && ../../../configure \
+	  --enable-rpm \
+	  --prefix=/usr \
+	  --enable-download \
+	  --enable-build-libraries="factory pari flint ntl fflas_ffpack memtailor mathic mathicgb";
```

Comment:
Perhaps I was using your Makefile incorrectly, but without any extra arguments, those submodules fail to compile; for instance:
```
checking whether the C++ compiler works... no
configure: error: in `/home/mahrud/Documents/Projects/M2/M2/M2/BUILD/dan/builds.tmp/galois-master/submodules/fflas_ffpack':
configure: error: C++ compiler cannot create executables
See `config.log' for more details
make[1]: *** [GNUmakefile:171: submodules/fflas_ffpack/Makefile] Error 77
```
Some of them are easy to fix manually then continue making again, but at some point, I ran into pretty terrible errors related to thread-local storage in Flint. I suspect this could be only on Fedora 26.